src/bundle/Resources/public/js/scripts/sidebar/main.menu.js

```diff
@@ -0,0 +1,132 @@
+            const label = item.querySelector('.ibexa-main-menu__item-text-column').textContent;
+
+            if (firstLevelMenuNode.classList.contains('ibexa-main-menu__navbar--collapsed')) {
+                item.setAttribute('title', label);
+            } else {
+                item.removeAttribute('data-original-title');
+            }
+
+            eZ.helpers.tooltips.parse(mainMenuNode);
+            item.removeAttribute('title');
+        });
+
+        
+        eZ.helpers.tooltips.hideAll();
+    };
+    const addResizeListeners = ({ clientX }) => {
+        resizeStartPositionX = clientX;
+        secondLevelMenuNode.classList.add('ibexa-main-menu__navbar--resizing');
+        secondMenuLevelCurrentWidth = secondLevelMenuNode.getBoundingClientRect().width;
+
+        doc.addEventListener('mousemove', triggerSecondLevelChangeWidth, false);
+        doc.addEventListener('mouseup', removeResizeListeners, false);
+    };
+    const removeResizeListeners = () => {
+        doc.removeEventListener('mousemove', triggerSecondLevelChangeWidth, false);
+        doc.removeEventListener('mouseup', removeResizeListeners, false);
+        secondLevelMenuNode.classList.remove('ibexa-main-menu__navbar--resizing');
+    };
+    const triggerSecondLevelChangeWidth = ({ clientX }) => {
+        const resizeValue = secondMenuLevelCurrentWidth + (clientX - resizeStartPositionX);
+        const newMenuLevelWidth = resizeValue > SECOND_LEVEL_MANUAL_RESIZE_MIN_WIDTH ? resizeValue : SECOND_LEVEL_COLLAPSED_WIDTH;
+
+        localStorage.secondLevelMenuWidth = newMenuLevelWidth;
+        setWidthOfSecondLevelMenu();
+    };
+    const fitMenu = () => {
+        const diff = Math.max(HEADER_HEIGHT - global.scrollY, 0);
+
+        firstLevelMenuNode.querySelector('.ibexa-main-menu__items-list').style.top = `${diff}px`;
+        firstLevelMenuNode.querySelector('.ibexa-main-menu__items-list').style.height = `${global.innerHeight - diff}px`;
+
+        secondLevelMenuNode.querySelectorAll('.ibexa-main-menu__items-list').forEach((itemList) => {
+            itemList.style.top = `${diff}px`;
+            itemList.style.height = `${global.innerHeight - (diff + TOGGLER_HEIGHT)}px`;
+        });
+    };
+
+    fitMenu();
+    parseMenuTitles();
+    setWidthOfSecondLevelMenu();
+
+    global.addEventListener('scroll', fitMenu, false);
+    global.addEventListener('resize', fitMenu, false);
+
+    firstLevelMenuNode.querySelectorAll('.ibexa-main-menu__item-action').forEach((button) => {
+        button.addEventListener('click', showSecondLevelMenu, false);
+    });
+
+    secondLevelMenuNode.querySelector('.ibexa-main-menu__toggler').addEventListener('click', toggleSecondLevelMenu, false);
+    secondLevelMenuNode.querySelector('.ibexa-main-menu__resizer').addEventListener('mousedown', addResizeListeners, false);
+    secondLevelMenuNode.querySelectorAll('.ibexa-main-menu__tooltip-trigger').forEach((button) => parsePopup(button));
```

Comment:
```suggestion
    secondLevelMenuNode.querySelectorAll('.ibexa-main-menu__tooltip-trigger').forEach(parsePopup);
```
could be like this ;)